tests/test_cuda_bd_exchange_mover.py

```diff
@@ -352,7 +352,7 @@ def test_bd_exchange_deterministic_moves(proposals_per_move, precision, seed):
         (5000, 40000, 3.0),
         (10000, 250000, 3.0),
         # The 6.0nm box triggers a failure that would occur with systems of certain sizes, may be flaky in identifying issues
-        pytest.param(1, 10000, 6.0, marks=pytest.mark.nightly(reason="slow")),
+        pytest.param(1, 20000, 6.0, marks=pytest.mark.nightly(reason="slow")),
```

Comment:
Before changes
```
tests/test_cuda_bd_exchange_mover.py::test_moves_in_a_water_box[2023-float32-0.0001-0.002-1-10000-6.0]
Accepted 1 moves on step 3024
Accepted 1 moves on step 3666
Accepted 1 moves on step 3981
Accepted 1 moves on step 4869
Accepted 1 moves on step 6001
Accepted 1 moves on step 7153
Accepted 1 moves on step 7773
Accepted 1 moves on step 8244
Accepted 8 of 10000 moves
```
After separated noise
```
tests/test_cuda_bd_exchange_mover.py::test_moves_in_a_water_box[2023-float32-0.0001-0.002-1-20000-6.0]
Accepted 1 moves on step 10364
Accepted 1 moves on step 10394
Accepted 1 moves on step 17521
Accepted 1 moves on step 18071
Accepted 1 moves on step 19339
Accepted 5 of 20000 moves
```

Don't think this is a sign of bad behavior, rather not converged and the inherent randomness. Running the buckyball case as that is a more definitive analysis of this problem.